cmd/litcli/privacy_map.go

```diff
@@ -14,8 +14,8 @@ var privacyMapCommands = cli.Command{
 	Name:      "privacy",
 	ShortName: "p",
 	Usage: "Access the real-pseudo string pairs of the " +
-		"privacy mapper",
-	Category: "Privacy",
+		"privacy mapper.",
+	Category: "Firewall",
 	Flags: []cli.Flag{
```

Comment:
The first word of the sentence in "Usage" for the `realtopseudo` flag should be upper case.

Reply:
I changed the descriptions for oldest_first in actions and realtopseudo in privacy.